packages/documentation/copy/en/javascript/JSDoc Reference.md

```diff
@@ -463,25 +593,55 @@ function callbackForLater(e) {
 }
 ```
 
-## `@extends`
+# Documentation
+
+## `@deprecated`
+<div id="[deprecated-comments]"></div>
 
-When JavaScript classes extend a generic base class, there is nowhere to specify what the type parameter should be. The `@extends` tag provides a place for that type parameter:
+When a function, method, or property is deprecated you can let users know by marking it with a `/** @deprecated */` JSDoc comment. That information is surfaced in completion lists and as a suggestion diagnostic that editors can handle specially. In an editor like VS Code, deprecated values are typically displayed in a strike-through style ~~like this~~.
 
 ```js twoslash
-/**
- * @template T
- * @extends {Set<T>}
- */
-class SortableSet extends Set {
-  // ...
+// @noErrors
+/** @deprecated */
+const apiV1 = {};
+const apiV2 = {};
+
+apiV;
+// ^|
+
+
+```
+
+## `@see`
+
+`@see` lets you link to other names in your program:
+
+```ts twoslash
+type Box<T> = { t: T }
+/** @see Box for implementation details */
+type Boxify<T> = { [K in keyof T]: Box<T> };
+```
+
+Some editors will turn `Box` into a link to make it easy to jump there and back.
+
+## `@link`
+
+`@link` is like `@see`, except that it can be used inside other tags:
+
+```ts twoslash
+type Box<T> = { t: T }
+/** @return A {@link Box} containing the parameter. */
+function box<U>(u: U): Box<U> {
+  return { t: u };
 }
 ```
 
-Note that `@extends` only works with classes. Currently, there is no way for a constructor function extend a class.
+# Other
 
 ## `@enum`
 
 The `@enum` tag allows you to create an object literal whose members are all of a specified type. Unlike most object literals in JavaScript, it does not allow other members.
+`@enum` is intended for compatibility with Closure's `@enum` tag.
```

Comment:
Maybe call this Google Closure, I initially interpreted it as Clojure? :D

Reply:
Agree, most people haven't heard of Closure.